apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -203,6 +209,21 @@ router.post('/', function (req, res, next) {
         }
       });
     });
+  } else if (req.body.__action === 'unsafe_sharing_set_add') {
```

Comment:
This needs a feature flag check to disallow using this code if question sharing isn't enabled for a course.